app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    source_path = parts[1]
+    if len(parts) != 3:
+        raise ValueError("Invalid format")
+    destination_path = parts[2]
+    if not os.path.split(source_path) and destination_path.endswith("/"):
```

Comment:
The check is wrong

Reply:
Test this code with destination path that doesn't end with `/` - it will fail
We should allow such destination paths either.

And I don't understand why did you add this `not os.path.split(source_path)`